Scripts_to_load_tables/load_entities.js

```diff
@@ -0,0 +1,38 @@
+////// THIS SCRIPT LOADS THE INPUT JSON FILE INTO ENTITIES TABLE.THIS TABLE IS EXPOSED TO GROUP 2 THORUGH REST API'S //////////////////////////
+var fs = require("fs");
+var pg = require("pg");
+var content = fs.readFileSync("dbpedia_composer.json"); //296
+var jsonContent = JSON.parse(content);
+var cn = {
```

Comment:
Please move this logic to a global config file which is .gitignored but for which you provide a template https://github.com/MusicConnectionMachine/StructuredData/pull/24/files#r106035398